README.md

```diff
@@ -13,6 +13,15 @@ let krate: rustdoc_types::Crate = serde_json::from_str(&json_string)?;
 println!("the index has {} items", krate.index.len());
 ```
 
+For performance sensitive crates we suggest turning on the `rustc-hash`
+feature. This switches all data structures from `std::collections::HashMap` to
+`rustc-hash::FxHashMap` which improves performance when reading big JSON files
+(like `aws_sdk_rs`'s).
+
+We have tested this on `cargo-semver-checks` using `aws_sdk_ec2`'s JSON and
```

Comment:
I don't like the use of "We" here. I think it's important to have a clear line between what's maintained by the rust project, and what isn't.

Additionally I don't think the numbers presented here are particularly useful to people who don't know/care about csc internals. I think just a percentage improvement would much more useful.


> In testing on [cargo-semver-checks](some link), this provided a whatever% perfomance improvement on the `aws-sdk-ec2` crate.

Reply:
Reworded, I hope this is better?